seed/seed.json

```diff
@@ -192,10 +204,44 @@
             }
         },
         {
-            "name": "BraveAds.ShowCustomAdNotificationsOnNightlyChannelStudy",
+            "name": "BraveAds.ShowCustomAdNotificationStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 60,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "true"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 0,
+                    "feature_association": {
+                        "disable_feature": ["AdNotifications"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 40
+                }
+            ],
+            "filter": {
+                "min_version": "91.1.26.37",
+                "channel": ["NIGHTLY", "BETA"],
```

Comment:
Show custom ad notifications to 60% of users on Windows, macOS and Linux for both Beta and Nightly build channels